models/zone/src/zone-draft/presets/sample-data-fashion/with-country-australia.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { TZoneDraft } from '../../../types';
+import withCountryAustralia from './with-country-australia';
+
+describe('with the preset `country australia`', () => {
+  it('should return a zone with name `Australia`', () => {
+    const zone = withCountryAustralia().build<TZoneDraft>();
```

Comment:
we need tests for `buildGraphql` in this PR as well, as we'll eventually be making graphql requests 👍

Reply:
Thank you for saving me from myself 🎶

Reply:
@valoriecarli repossessing, sorry 😂